hack/make/build-rpm

```diff
@@ -70,10 +70,13 @@ set -e
 
 	builderDir="contrib/builder/rpm/${PACKAGE_ARCH}"
 	pkgs=( $(find "${builderDir}/"*/ -type d) )
-	if [ ! -z "$DOCKER_BUILD_PKGS" ]; then
-		pkgs=( $(echo ${DOCKER_BUILD_PKGS[@]/#/$builderDir\/}) )
-	fi
-	for dir in "${pkgs[@]}"; do
+    if [ ! -z "$DOCKER_BUILD_PKGS" ]; then
+        pkgs=()
+        for p in $DOCKER_BUILD_PKGS; do
+            pkgs+=( "$builderDir/$p" )
+        done
+    fi
+    for dir in "${pkgs[@]}"; do
```

Comment:
Oh man, I didn't notice this one before either, sorry!  These are spaces and should be tabs...  :disappointed: